backend/src/tests/user.test.ts

```diff
@@ -18,17 +16,51 @@ import {
     selectProjectsbyUserID,
 } from '../db/queries';
 import { registerKey } from '../services/registerKeyService';
+import { TokenPayload } from '../types/TokenPayload';
+import {
+    initializeUsers,
+    testUserName,
+    testUserPassword,
+    getUserToken,
+} from './../services/testService';
+import crypto from 'crypto';
```

Comment:
crypto could be switched out to uuidv4 to be more uniform with the rest of the codebase, but unlike frontend, it's not a big deal here.